lib/jarvis/workpool.rb

```diff
@@ -0,0 +1,52 @@
+require "concurrent"
+
+class WorkPool
+  ADMINISTRATIVE = :ADMINISTRATIVE
+  NORMAL = :NORMAL
+
+  class InvalidWorkPoolName < StandardError; end
```

Comment:
Can we add a base class for all our common errors? I think we might add more to the code base.